frontend/projects/abtesting/src/app/core/experiments/store/experiments.model.ts

```diff
@@ -163,10 +163,7 @@ export interface ExperimentPaginationParams {
 }
 
 export interface IContextMetaData {
-  appContext: string[],
-  expPoints: string[],
-  expIds: string[],
-  groupTypes: string[]
+  contextMetadata: object[];
```

Comment:
We have proper structure of contextMetadata so let create an interface and use that instead of generic one

Reply:
You can use something like this
```
interface IContextMetadata {
    EXP_IDS: string[],
    EXP_POINTS: string[],
    GROUP_TYPES: string[]
}

interface IContextMetadataContainer {
    [key: string]: IContextMetadata
}
```